docs/cosmology/index.rst

```diff
@@ -51,18 +60,16 @@ transverse proper kiloparsecs (kpc) corresponding to an arcminute at z=3::
   >>> cosmo.kpc_proper_per_arcmin(3)  # doctest: +FLOAT_CMP
   <Quantity 472.97709620405266 kpc / arcmin>
 
-Here WMAP9 is a built-in object describing a cosmology with the
-parameters from the nine-year WMAP results. Several other built-in
-cosmologies are also available (see `Built-in Cosmologies`_). The
-available methods of the cosmology object are listed in the methods
-summary for the `~astropy.cosmology.FLRW` class. If you are using
-IPython you can also use tab completion to print a list of the
-available methods. To do this, after importing the cosmology as in the
-above example, type ``cosmo.`` at the IPython prompt and then press
-the tab key.
+Here |WMAP9| is a built-in object describing a cosmology with the parameters
+from the nine-year WMAP results. Several other built-in cosmologies are also
+available (see `Built-in Cosmologies`_). The available methods of the cosmology
+object are listed in the methods summary for the |FLRW| class. If you are using
+IPython you can also use tab completion to print a list of the available
+methods. To do this, after importing the cosmology as in the above example,
```

Comment:
I don't think one should be mixing in general introduction to (i)python here.  I'd delete "If you are ... tab key."

Reply:
Agreed.